etna/metrics/metrics.py

```diff
@@ -295,7 +306,8 @@ def __init__(self, mode: str = MetricAggregationMode.per_segment, **kwargs):
         kwargs:
             metric's computation arguments
         """
-        super().__init__(mode=mode, metric_fn=max_deviation, **kwargs)
+        max_deviation_per_output = partial(max_deviation, multioutput="per_output")
```

Comment:
May be change the naming of multioutput cases to sklearn ones?  per_output -> raw_values?

Reply:
Ok, we could do that. I don't really like the naming "raw_values", because it doesn't mean anything  for me, but it could be more intuitive for sklearn users.

Value of "joint" isn't the same as "uniform_average", are we going to do smth with it?

Reply:
Why it isn't the same? mode= joint should work line sklearn default

Reply:
Value "uniform average" is sklearn's default. It takes all the scores across different outputs and finds its mean value. 

Value "joint" preserves current default behaviour in etna. It works as if all matrix is flattened and one value is calculated.

So, this types produce different results for metrics that aren't calculated as mean value of some errors. For example, `max_deviation`, `wape`.